### PR TITLE
Remove commented out code

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
We have version control systems for a reason, if we want code to not run it should be removed. I personally see no value in keeping these around. I presume at one point they were spamming debug.log so we commented them out, but we really should have just removed them.

I believe all of this is dash specific code but any conflicts this does create are so minor they are not of concern imo.

Signed-off-by: Pasta <pasta@dashboost.org>

### CODE REVIEW

File: src/keepass.cpp [hunk 1]
```diff
@@ -341,7 +341,6 @@ void CKeePassIntegrator::doHTTPPost(const std::string& sRequest, int& nStatusRet
     evhttp_add_header(output_headers, "Connection", "close");
 
     // Logging of actual post data disabled as to not write passphrase in debug.log. Only enable temporarily when needed
```
Comment (codablock) [inline src/keepass.cpp]: Maybe also remove this comment as it was referring to the commented out log print

File: src/keepass.cpp [hunk 2]
```diff
@@ -452,7 +451,6 @@ std::vector<CKeePassIntegrator::CKeePassEntry> CKeePassIntegrator::rpcGetLogins(
     doHTTPPost(request.getJson(), nStatus, strResponse);
 
     // Logging of actual response data disabled as to not write passphrase in debug.log. Only enable temporarily when needed
```
Comment (codablock) [inline src/keepass.cpp]: same

File: src/miner.cpp [hunk 3]
```diff
@@ -224,8 +224,6 @@ std::unique_ptr<CBlockTemplate> BlockAssembler::CreateNewBlock(const CScript& sc
     // Update coinbase transaction with additional info about masternode and governance payments,
     // get some info back to pass to getblocktemplate
     FillBlockPayments(coinbaseTx, nHeight, blockReward, pblocktemplate->voutMasternodePayments, pblocktemplate->voutSuperblockPayments);
-    // LogPrintf("CreateNewBlock -- nBlockHeight %d blockReward %lld txoutMasternode %s coinbaseTx %s",
-    //             nHeight, blockReward, pblocktemplate->txoutsMasternode.ToString(), coinbaseTx.ToString());
```
Comment (codablock) [inline src/miner.cpp]: This probably wouldn't even compile anymore :see_no_evil:

**Comment by PastaPastaPasta** [comment]
resolved @codablock's suggestions, also removed some extra code in keepass.cpp which was commented out

**Pull request merged**